plugins/logger-json/src/templates/logger.config.ts

```diff
@@ -3,21 +3,98 @@ import { Params } from "nestjs-pino";
 
 ADDITIONAL_LOG_PROPERTIES_KEY;
 
+const defaultSerializer = (
```

Comment:
although this implementation is really smart, I'd recommend to simplify it by leveraging the nestjs-pino library configuration that allow to achieve the same by configuration. Check here for more details https://getpino.io/#/docs/redaction

Reply:
Done @overbit.